Issue1〜3の修正

masa555555 opened the pull request:
Issue1〜3の解消

masa555555 committed:
Issue1

masa555555 committed:
Issue2

masa555555 committed:
Issue3

yuevo commented on app/controllers/calendars_controller.rb:
```diff
@@ -2,7 +2,7 @@ class CalendarsController < ApplicationController
   # １週間のカレンダーと予定が表示されるページ
   def index
-    getWeek
+    getweek
```
```
getweek
```
こちらは、キャメルケース からスネークケースに変更したいところになります。
スネークケースとは「item_get」のように単語同士をアンダーバーで繋ぐ記述方法です。
これを参考にし、修正してみましょう！

yuevo commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,7 +18,7 @@ def plan_params
   end
 
-  def getWeek
+  def getweek
```
```
getweek
```
こちらは、キャメルケース からスネークケースに変更したいところになります。
スネークケースとは「item_get」のように単語同士をアンダーバーで繋ぐ記述方法です。
これを参考にし、修正してみましょう！

yuevo reviewed (commented):
レビューのご依頼ありがとうございます！
２点コメントさせていただいました。
ご確認のほど、よろしくお願いいたします！

masa555555 committed:
Issue2 修正

masa555555 commented:
修正しました

yuevo commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +34,7 @@ def getWeek
       plans.each do |plan|
```
差分が出ておりませんので、こちらから失礼します。
5行目と21行目の記述が修正前に戻ってしまっております。
`キャメルケース`は主にクラス名、
`スネークケース`は主に変数やメソッド名に使用されます。

よって、上記の２カ所はメソッド名なので、スネークケースの記述方法に修正しましょう！

yuevo reviewed (commented):
レビューのご依頼ありがとうございます！
１点コメントさせていただいました。
ご確認のほど、よろしくお願いいたします！

masa555555 committed:
issue2 修正

what
①メソッドをキャメルケースからスネークケースに修正、
②getWeekからget_weekに修正
why
命名規則の遵守できていなかったので、修正しました

yuevo commented:
レビュー依頼ありがとうございます！
LGTMとさせていただきます。実装お疲れ様でした！
こちらのプルリクエストは忘れずにマージしておきましょう。
引き続き実装頑張っていきましょう！
